Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust
+
+RUN apt-get update && \
+    apt-get install -y cmake && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY . /opt/tarpaulin/
+
+RUN cd /opt/tarpaulin/ && \
+    cargo install && \
+    rm -rf /opt/tarpaulin/ && \
+    rm -rf /usr/local/cargo/registry/
+
+WORKDIR /volume
+
+CMD cargo build && cargo tarpaulin
```

Comment:
I think the `cargo build` is not necessary, but I've added it for now. See #67 for why.